core/src/Label/Label.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import styles from "./Label.less";
+
+export const Label = ({ label, name, datatype, required }) => (
+  <label className={styles.Label}>
+    {label}{" "}
```

Comment:
Lines 7-11 should not be determined here. The design-system needs to be abstracted enough that it can be applied in multiple contexts. Rendering schema information is an app detail specific to the content-editor. This should be a per-calculated string, from within manager-app, that is rendered by the label component.

Reply:
Label also needs to act as a HOC. By having a `label` HTMLElement wrap an `input` HTMLElement you get input focus when clicking on the label. This can be a desired experience in many places.

Reply:
![image](https://user-images.githubusercontent.com/26661451/51707323-c11e6300-1fd5-11e9-8725-e246bbd718e2.png)

Reply:
Label now looks for props and determines wether or not to render context based on that.

Reply:
I would like this to be a higher order component. Usage should look like this.

```
<FieldComponent>
  <Label>
    Label child tree [{name}] {datatype}
  </Label>

</FieldComponent>
```
This way in the future we can continue to provide unique experience with label usage without the need to continually modify this component.